### ISSUE TITLE
Status

### ISSUE THREAD

**Issue by samssann** [tqdm/tqdm.cpp#44]
This is dead right? That's a pity.. seemed like a very useful project. C++ has lousy progress bar libs

**Comment by casperdcl** [comment]
not dead, just blocked on [tqdm/tqdm v5.0.0](https://github.com/tqdm/tqdm/milestone/8) (specifically, tqdm/tqdm#198)

**Comment by samssann** [comment]
Thanks for the info @casperdcl . Hopefully that gets sorted out sooner or later. Appreciate your work 👍

**Comment by HsuanTingLu** [comment]
Since tqdm/tqdm#198 has been sorted out,
is it safe to say that the submodule architectures are stable now?

**Comment by casperdcl** [comment]
Well I partially fixed https://github.com/tqdm/tqdm/issues/198 and split out additional steps into https://github.com/tqdm/tqdm/issues/815 ... I really don't know when I'll ever get around to fixing it.

Honestly, given that I get no funding and even less time, I just fix things which I personally need to use and just about keep up with responding to issues/PRs. I haven't needed a progressbar in C++ for the last couple of years so haven't pushed this :/

**Comment by HsuanTingLu** [comment]
Fair enough, though some directions or guides would be nice to have,
but still, thank you, tqdm rocks!
